edge-apis/credentials.go

```diff
@@ -109,15 +124,43 @@ func (c *BaseCredentials) GetCaPool() *x509.CertPool {
 	return c.CaPool
 }
 
+// Method provides a base implementation to return the authentication method of a Credentials instance.
+func (c *BaseCredentials) Method() string {
```

Comment:
This method was purposefully not implemented as it stops its usage as a `Credentials` interface implementor. The BaseCredentials struct is meant to be embedded as a component of another struct that provides that method and fulfills the interface.

Reply:
I've removed it. It doesn't need to be an interface so long as the SDK user knows to update credentials after the `NewContext` call.